index.php

```diff
@@ -1,17 +1,19 @@
 <?php
 
-require_once 'src/Watch.php';
 require_once 'src/WatchModel.php';
 require_once 'src/WatchViewHelper.php';
 
-
 $db = new PDO('mysql:host=db; dbname=watch_collection', 'root', 'password');
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 
 $watchesModel = new WatchModel($db);
 $watches = $watchesModel->getAllWatches();
 
-
+if(isset($_POST['watch_id'])) {
+    $watch = new WatchModel($db);
```

Comment:
No need to have a second WatchModel - you can reuse the $watchesModel that you created above